website/docs/topics/feature-flags/feature-flag-best-practices.md

```diff
@@ -1,339 +1,188 @@
-
-## 1. Enable run-time control. Control flags dynamically, not using config files.
+These principles are:
 
-A scalable feature management system evaluates flags at runtime. Flags are dynamic, not static. If you need to restart your application to turn on a flag, you are using configuration, not feature flags.
+1. [Enable runtime control: Control flags dynamically instead of through config files.](#1-enable-runtime-control)
+2. [Never expose PII: Follow the principle of least privilege.](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices)
+3. [Reduce latency: Evaluate flags as close to the user as possible.](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices)
+4. [Scale horizontally: Decouple read and write operations.](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices)
+5. [Limit payload size: Keep feature flag payload lean.](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices)
+6. [Design for failure: Prioritize availability over consistency.](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices)
+7. [Use short-lived flags: Do not confuse flags with application configuration.](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices)
+8. [Use unique flag names: Enforce naming conventions.](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices)
+9. [Choose open by default: Democratize feature flag access.](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices)
+10. [Do no harm: Prioritize consistent user experience.](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices)
+11. [Enable traceability: Make flag evaluation transparent and testable.](https://docs.getunleash.io/topics/feature-flags/feature-flag-best-practices)
 
-A large-scale feature flag system that enables runtime control should have at minimum the following components:
+Let’s dive deeper into each principle.
 
-**1. Feature Flag Control Service**: Use a centralized feature flag service that acts as the control plane for your feature flags. This service will handle flag configuration. The scope of this service should reflect the boundaries of your organization.
 
-Independent business units or product lines should potentially have their own instances, while business units or product lines that work closely together should most likely use the same instance in order to facilitate collaboration. This will always be a contextual decision based on your organization and how you organize the work, but keep in mind that you’d like to keep the management of the flags as simple as possible to avoid the complexity of cross-instance synchronization of feature flag configuration.
+## 1. Enable runtime control
```

Comment:
@riboflavin was there a particular reason why you went for this annotation in your PR: `1\.`? Is it because the space between the number and the text is rendered too small?

Reply:
I can't remember exactly why I made this change, but I think you should fix it up the way you think is best @melindafekete!